doc/chapter-configuration.asciidoc

```diff
@@ -147,6 +147,26 @@ Available dialogs are:
 * `dirbrowser`
 * `searchresultslist`
 
+=== New key bindings
+
+You can bind sequences of keys to lists of commands with the <<bind,`bind`>> configuration command.
+This combines the posibilites of <<bind-key,`bind-key`>> (option to have single-key bindings) with the posibilies of <<macro,`macros`>> (option to bind multiple commands).
+
+The syntax for a key binding looks like this:
+
+    bind <key-sequence> <dialogs> <command list> -- "<description>"
+
+The key-sequence can contain a single key, or a combination of keys to be pressed in order.
+Alphanumeric keys are written as-is (e.g. using `a` for the kbd:[A] key, and `A` for the kbd:[Shift+A] key combination).
+Special keys like kbd:[ENTER], kbd:[UP], and function keys are specified within angle brackets (e.g. `<ENTER>`, `<UP>`, and `<F1>`).
+The full list of special keys can be found in the "Keys" subsection above.
```

Comment:
A hyperlink to that subsection would be very useful, I reckon. Especially if somebody gets an anchored URL to this subsection and wants to follow it. Same down below with the "Dialog" subsection.